modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOptionFacet.testcase

```diff
@@ -14,6 +14,42 @@ definition {
 		CommerceConfiguration.commerceTearDown();
 	}
 
+	@description = "This is a test for COMMERCE-10454."
+	@priority = 3
+	test CanViewProductSourceUpdateWhenChangedFromDynamicToManual {
+		task ("Given the Product Publisher is added") {
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/minium");
+
+			CommerceAcceleratorsInitializer.addPortlet(portletName = "Product Publisher");
+
+			Portlet.gotoPortletOptions(
+				portletName = "Product Publisher",
+				portletOption = "Configuration");
+		}
+
+		task ("And the Product Entries is not shown") {
+			SelectFrameTop();
+
+			SelectFrame.selectFrameNoLoading(locator1 = "IFrame#CONFIGURATION");
+
+			AssertElementNotPresent(
+				key_panel = "Product Entries",
+				locator1 = "CommerceEntry#CONFIGURATION_PANEL_EXPANDED");
+		}
+
+		task ("When the Manual option is selected") {
+			Click(
+				locator1 = "Radio#ANY",
+				radioLabel = "Manual");
+		}
+
+		task ("Then the Product Entries is automatically shown") {
```

Comment:
Can you fix it into "Then the Product Entries are automatically shown"?

Thanks!